main.go

```diff
@@ -100,6 +110,12 @@ func main() {
 			Usage:  "Private key used in client TLS auth",
 			Value:  "",
 		},
+		cli.StringFlag{
+			EnvVar: "MACHINE_SSH_CLIENT_TYPE",
+			Name:   "ssh-client-type",
+			Usage:  "Type of SSH client to use.  Options: gonative, shellout",
+			Value:  "",
```

Comment:
Hm, so, the reason I didn't just default this to `shellout`, was because we do a check to see if there is an `ssh` binary around to determine if we should use `shellout` or `gonative`.  If the user doesn't have `ssh` I would really like running the commands to be transparent to them anyway without them having to worry about setting the value.  Maybe I could do the check for the binary in the `Before` block instead of in the `ssh` package?